### PR TITLE
Fix visibility of action buttons for Isos and templates

### PR DESCRIPTION
**Pull request by Pearl1594**
Despite the template / iso being in 'Ready' state i.e., downloaded, the action buttons (copy and delete) don't appear even in refresh
image

After fix:
image

image

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan package

File: src/views/image/IsoZones.vue [hunk 1]
```diff
@@ -32,15 +32,15 @@
...
+            :disabled="!('copyIso' in $store.getters.apis && record.isready)"
             icon="copy"
             shape="circle"
             :loading="copyLoading"
             @click="showCopyIso(record)" />
         </span>
         <span style="margin-right: 5px">
           <a-popconfirm
-            v-if="'deleteIso' in $store.getters.apis"
+            v-if="'deleteIso' in $store.getters.apis && record.isready"
```
Comment (rhtyd) [inline src/views/image/IsoZones.vue:43]: @Pearl1594 can you check against old UI, if deleting should be allowed? For example for error on download or uploading of local file?

File: src/views/image/TemplateZones.vue [hunk 2]
```diff
@@ -32,15 +32,15 @@
...
+            :disabled="!('copyTemplate' in $store.getters.apis && record.isready)"
             icon="copy"
             shape="circle"
             :loading="copyLoading"
             @click="showCopyTemplate(record)" />
         </span>
         <span style="margin-right: 5px">
           <a-button
-            :disabled="!('deleteTemplate' in $store.getters.apis)"
+            :disabled="!('deleteTemplate' in $store.getters.apis && record.isready)"
```
Comment (rhtyd) [inline src/views/image/TemplateZones.vue:43]: @Pearl1594 can you check against old UI, if deleting should be allowed? For example for error on download or uploading of local file?

**Review by davidjumani — APPROVED** [review]
LGTM

File: src/views/image/TemplateZones.vue (hunk 2 above)
Comment (rhtyd) [reply src/views/image/TemplateZones.vue]: ping @Pearl1594

Comment (Pearl1594) [reply src/views/image/TemplateZones.vue]: @rhtyd the legacy UI allows deletion of templates / ISOs incase there is a failure in downloading it

**Review by rhtyd — APPROVED** [review]

**Pull request merged**